packages/lesswrong/components/posts/PostsPage/PostActions.jsx

```diff
@@ -140,6 +140,16 @@ class PostActions extends Component {
             Edit
           </MenuItem>
         </Link>}
+        { Users.canCollaborate(currentUser, post) && 
+          <Link to={{pathname:'/collaborateOnPost/', search:`?${qs.stringify({postId: post._id})}`}}>
```

Comment:
URL shouldn't have a trailing slash.
```suggestion
          <Link to={{pathname:'/collaborateOnPost', search:`?${qs.stringify({postId: post._id})}`}}>
```